shell/platform/common/cpp/text_input_model.cc

```diff
@@ -90,12 +161,15 @@ bool TextInputModel::Backspace() {
   if (DeleteSelected()) {
     return true;
   }
-  // If there's no selection, delete the preceding codepoint.
+  // If there's no selection; delete the preceding codepoint.
```

Comment:
s/If there's/There is/ to make the grammar correct with the ;

Reply:
Done here and below.